rules/aws_cloudtrail_rules/aws_ec2_traffic_mirroring.py

```diff
@@ -0,0 +1,48 @@
+from panther_base_helpers import aws_rule_context
+
+
+def rule(event):
+    # Return True to match the log event and trigger an alert.
+    event_names = [
+        "CreateTrafficMirrorFilter",
+        "CreateTrafficMirrorFilterRule",
+        "CreateTrafficMirrorSession",
+        "CreateTrafficMirrorTarget",
+        "DeleteTrafficMirrorFilter",
+        "DeleteTrafficMirrorFilterRule",
+        "DeleteTrafficMirrorSession",
+        "DeleteTrafficMirrorTarget",
+        "DescribeTrafficMirrorFilters",
+        "DescribeTrafficMirrorSessions",
+        "DescribeTrafficMirrorTargets",
+        "ModifyTrafficMirrorFilterNetworkServices",
+        "ModifyTrafficMirrorFilterRule",
+        "ModifyTrafficMirrorSession",
+    ]
+    return (
+        event.get("eventSource", "") == "ec2.amazonaws.com"
+        and event.get("eventName", "") in event_names
+    )
+
+
+def title(event):
+    # (Optional) Return a string which will be shown as the alert title.
+    # If no 'dedup' function is defined, the return value of this method will
+    # act as deduplication string.
+    return (
+        f"{event.get('userIdentity',{}).get('type','no-type')} ec2 activity found for "
```

Comment:
this should likely be `userIdentity.arn` ( instead of type ).

Reply:
is there any particular reason we'd want userIdentity.arn over userIdentity.principleId? i agree that there's a more suitable value than type and am curious about the benefits of having the whole arn in the title vs the principle id!